packages/amplify-function-plugin-interface/src/index.ts

```diff
@@ -55,7 +55,7 @@ export type BuildRequest = {
   env: string;
   srcRoot: string;
   runtime: string;
-  legacyBuildHookParams?: {
+  legacyBuildHookParams: {
```

Comment:
Is this tied to the python provider or the platform?

Reply:
Platform, but it's fine either way